### ISSUE TITLE
Plastic is currently unavailable to users of the open source vscode

### ISSUE THREAD

**Issue by plul** [will-stone/plastic#98]
Hi, I have used plastic for a while now, but recently I could not find the extension when searching for it in vscode.
Because I am using the open source version of code, plastic (among other extensions) does not show up in the extension list.

The reason for this is due to Microsoft licensing. The Arch Linux wiki [explains it better](https://wiki.archlinux.org/index.php/Visual_Studio_Code#No_extensions_found)

In short, consider uploading the extension to the [Open VSX Registry](https://open-vsx.org/). This would make it available to users of the open source vscode.

**Comment by will-stone** [comment]
Hi @plul,

Alas, I'm unlikely to ever use Open VSX and so this can be considered out-of-scope for this project. You could clone the project to your Extensions folder. I do this when developing Plastic.

**Comment by plul** [comment]
That's perfectly understandable. The straightforward workaround as suggested on the Arch wiki is to manually download and install the .vsix file from the web marketplace page.
